packages/decision-tree/src/components/Question/questionTypes/MultipleChoice/MultipleChoice.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from 'react';
+import { type QuestionProps } from '@components/Question/types';
+import { Box, Checkbox } from '@cmpsr/components';
+import { QuestionTitle } from '../../components/QuestionTitle';
+import { MultipleChoiceQuestion } from './types';
+import { HandleAnswersActions } from '@hooks';
+
+export const MultipleChoice: FC<QuestionProps> = ({ data, answersDispatch }) => {
```

Comment:
unit tests?

Reply:
done in Question.test.ts, should I move it on each type of question? I just thought it would make more sense to have 1 test file rather than 5.